src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs

```diff
@@ -274,7 +274,7 @@ private CompilationModules GetCompileModules(CompilationTarget target)
             {
                 compilation = compilation.AddSyntaxTrees(new[]
                 {
-                    CSharpSyntaxTree.ParseText(newAttributes, parseOptions)
+                    CSharpSyntaxTree.ParseText(newAttributes, parseOptions, nameof(ApplyProjectInfo) + ".cs", Encoding.UTF8)
```

Comment:
two nits:
1. Use string interpolation
2. specify the name of the option parameters like at https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Dnx.Compilation.CSharp/RoslynCompiler.cs#L343